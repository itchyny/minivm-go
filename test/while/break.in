a = 1
while true
  print a
  if a >= 10
    break
  end
  a = a + 1
end

print 100

b = 1
while b < 10
  c = 1
  while c < 10
    print b * c
    if c > 4
      break
    end
    print c
    c = c + 1
  end
  if b > 4
    break
  end
  print b
  b = b + 1
end
