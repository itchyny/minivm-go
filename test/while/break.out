1
2
3
4
5
6
7
8
9
10
100
1
1
2
2
3
3
4
4
5
1
2
1
4
2
6
3
8
4
10
2
3
1
6
2
9
3
12
4
15
3
4
1
8
2
12
3
16
4
20
4
5
1
10
2
15
3
20
4
25
